tools/tests/checkbox/test.js

```diff
@@ -42,6 +43,42 @@ describe('Checkbox component', () => {
         click(checkbox);
         assert(checkMark.style.display === 'none', 'Check mark is not visible when the checkbox is selected.');
     });
+
+    it('Should update it\'s state if the checked attribute gets changed', () => {
+        const checkbox = document.getElementsByTagName('gameface-checkbox')[0];
+        assert(checkbox.checked === false, 'Checkbox is checked by default.');
+
+        checkbox.setAttribute('checked', '');
+
+        assert(checkbox.checked === true, 'Checkbox should have property checked equal to true.');
+        assert(checkbox.hasAttribute('checked') === true, 'Checkbox should have property checked.');
+    });
+
+    it('Should update it\'s state if the checked property gets changed', () => {
+        const checkbox = document.getElementsByTagName('gameface-checkbox')[0];
+        assert(checkbox.checked === false, 'Checkbox is checked by default.');
+
+        checkbox.checked = true;
+        assert(checkbox.hasAttribute('checked') === true, 'Checkbox should have property checked attribute.');

```

Comment:
```suggestion
        assert(checkbox.checked === true, 'Checkbox should have property checked equal to true.');
        assert(checkbox.hasAttribute('checked') === true, 'Checkbox should have property checked attribute.');
```

Reply:
We need to check that as well so we test if the state of the checked property is changed correctly